docs_src/microservices/application/AdvancedTopics.md

```diff
@@ -500,4 +500,7 @@ func(sdk *appsdk.AppFunctionsSDK, serviceKey string) (string, error) {
 ```
 !!! note Known Service Keys
     Service keys for known EdgeX services can be found under clients in [go-mod-core-contracts](https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/clients/constants.go#L58-L72)
-		
\ No newline at end of file
+		
+### Stopping the Service
+
+Application Services will listen for SIGTERM / SIGINT signals from the OS and attempt to cleanly exit in response.  They can also be stopped programmatically by calling `sdk.MakeItStop()` on the running SDK instance.
```

Comment:
`MakeItStop` doesn't exit the service, just the function pipeline. This should be clarifies and provide example use case when you would use it in your custom App Service.